sunxspex/sunxspex_fitting/stix_spec_code.py

```diff
@@ -40,8 +40,8 @@ def _get_spec_file_info(spec_file):
     _plus_half_bin_width = np.ceil(time_deltas/2)
     t_hi = times_mids + _plus_half_bin_width
 
-    spec_stimes = [Time(sdict["0"][0]["DATE_BEG"], format='isot', scale='utc')+TimeDelta(time_diff_so2e * u.s)+TimeDelta(dt * u.ds) for dt in t_lo]
-    spec_etimes = [Time(sdict["0"][0]["DATE_BEG"], format='isot', scale='utc')+TimeDelta(time_diff_so2e * u.s)+TimeDelta(dt * u.ds) for dt in t_hi]
+    spec_stimes = [Time(sdict["0"][0]["DATE-BEG"], format='isot', scale='utc')+TimeDelta(time_diff_so2e * u.s)+TimeDelta(dt * u.s) for dt in t_lo]
+    spec_etimes = [Time(sdict["0"][0]["DATE-BEG"], format='isot', scale='utc')+TimeDelta(time_diff_so2e * u.s)+TimeDelta(dt * u.s) for dt in t_hi]
```

Comment:
I'm not sure this change is needed if the goal is to get the value in seconds `to_value(u.s)` could be used and would remove the somewhat arbitrary `* (10**(-2))` below.

Reply:
Good catch! The `_ds_times2s_times`  function could be removed altogether, this seems like a relic from when I was coding for multiple file structures. Would it be better to just convert `times_mids` and `time_deltas` (lines 31 and 32, respectively) on the same lines and remove the `_ds_times2s_times` function? Say change line 31 to something like `time_mids = (sdict["2"][1]["time"]<<u.ds).to_value(u.s)` and change line 32 in a similar way?

Reply:
Oh yeah, that's a good point. Thanks for raising that change. I have just corrected that (the time is in units of cs) by multiplying tds by u.cs and keeping time_mids and time_deltas without units (assigning units to the arrays breaks the code if you try to select an event/background time region).